Puppet/Puppet/MSAnalyticsViewController.m

```diff
@@ -55,6 +55,8 @@ - (BOOL)shouldAutorotateToInterfaceOrientation:(UIInterfaceOrientation)toInterfa
 }
 
 - (IBAction)trackEvent {
+  [MSAnalytics trackEvent:self.eventName.text withProperties:self.properties];
```

Comment:
is this intentional?

Reply:
Is this intentional?

Reply:
Offline discussion - yes, it was intentional, but it prevents us from testing the scenario where both transmission targets are enabled but we want the runtime target to override the default target.  This was enabled in the previous version of the test app but is not in this one.  

We are going to create a card to improve the test app UI and functionality to 1) reenable this scenario and 2) be more easily understood.  Right now it's kind of tribal knowledge which buttons have to be pressed to get certain behavior, and we should try to make it more clear.